src/mscorlib/src/System/Globalization/GlobalizationMode.cs

```diff
@@ -7,6 +7,27 @@ namespace System.Globalization
     internal sealed partial class GlobalizationMode
     {
         private const string c_InvariantModeConfigSwitch = "System.Globalization.Invariant";
+        // Linux doesn't support environment variable names including dots
+        private const string c_InvariantModeEnvironmentVariable = "System_Globalization_Invariant";
```

Comment:
We need to get a standardized convention for the environment variables. We have `COMPlus_...`, `DOTNET_...`, ...  it is all over the place.

Is there any other prior art that is using `System_...` prefix?

Reply:
I agree we need to standardize.  And DOTNET_ seems like the best choice, as from a consumer's perspective their really shouldn't be a difference between coreclr, corefx, etc.  Most of the existing environment variables are COMPlus_... would there be any concerns changing them?

Reply:
DOTNET_ prefix seam reasonable to me. if all agree, I can change the name here. we can log an issue too to track renaming the other introduced environment variable.

P.S. I don't like COMPLUS_ nor CORCLR _ prefixes as most of people don't know what is complus and also CORECLR can give the impression this is specific to coreclr repo (and never apply for corert for instance)